ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -55,19 +55,21 @@ install_mofed() {
     stream=false
     gversion="$VERSION_ID"
     if [ "$gversion" == "8" ]; then
-        gversion="8.6"
+        # Mellanox does not have a release for 8.9 yet.
+        gversion="8.8"
```

Comment:
We are not testing 8-stream in CI, so this check is not needed.
Otherwise we need to pick the newest 8.x or 9.x release that has MOFED drivers for the version we specify.
So unless we also are specifying "latest" for MLNX_VER_NUM, the is a chance the install will not work.

This does not have to be changed now.